BowlingBall.Tests/GameFixture.cs

```diff
@@ -1,14 +1,165 @@
 using System;
 using Xunit;
+using BowlingBall;
+//using Moq;
 
 namespace BowlingBall.Tests
 {
     public class GameFixture
     {
+
+        [Fact]
+        public void Game_when_UserRollsValidInput_should_ReturnCorrectScore()
+        {
+            Game game = new Game();
+            int[] pins = { 1, 2, 3, 4 };
+
+            game.Roll(pins);
+
+            Assert.Equal(10, game.GetScore());
+
+        }
+
+        [Fact]
+        public void Game_when_UserRollsAllZeroes_should_ReturnCorrectScore()

```

Comment:
`ReturnCorrectScore` is not a meaningful. What is the definition of `CorrectScore`?
 
`Game_when_UserRollsAllZeroes_should_ReturnScoreZero`

rename for all